src/auditlog/migrations/0001_initial.py

```diff
@@ -24,8 +24,8 @@ class Migration(migrations.Migration):
                 ('action', models.PositiveSmallIntegerField(verbose_name='action', choices=[(0, 'create'), (1, 'update'), (2, 'delete')])),
                 ('changes', models.TextField(verbose_name='change message', blank=True)),
                 ('timestamp', models.DateTimeField(auto_now_add=True, verbose_name='timestamp')),
-                ('actor', models.ForeignKey(related_name='+', on_delete=django.db.models.deletion.SET_NULL, verbose_name='actor', blank=True, to=settings.AUTH_USER_MODEL, null=True)),
-                ('content_type', models.ForeignKey(related_name='+', verbose_name='content type', to='contenttypes.ContentType')),
+                ('actor', models.ForeignKey(on_delete=django.db.models.deletion.SET_NULL, to=settings.AUTH_USER_MODEL, related_name='+', verbose_name='actor', blank=True, null=True)),
+                ('content_type', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='contenttypes.ContentType', related_name='+', verbose_name='content type')),
```

Comment:
Any reason this migration is edited?

Reply:
1. `content_type` requires `on_delete` kwargs in django 2.0 (see [Django 2.0 changelog](https://docs.djangoproject.com/en/2.0/releases/2.0/#features-removed-in-2-0))

```
File "/home/rod/src/git/django-auditlog.git/src/auditlog/migrations/0001_initial.py", line 30, in Migration
    ('content_type', models.ForeignKey(related_name='+', verbose_name='content type', to='contenttypes.ContentType')),
TypeError: __init__() missing 1 required positional argument: 'on_delete'
⋮
ERROR:   py34-django-20: commands failed
ERROR:   py35-django-20: commands failed
ERROR:   py36-django-20: commands failed
```
2. `actor` changes not intentional, so have reverted that line (think that might've been how it was auto generated by `manage.py makemigrations`, which I copied-pasted =)

Django 2.0 changelog suggests squashing migrations to manage this, and I'd support that approach.

Reply:
Thanks for the explanation. I'm also for squashing migrations, however, I'm not sure about the implications of squashing for existing installs. Should be a separate issue anyways I guess.